masternodes/setup.rst

```diff
@@ -269,10 +269,12 @@ Then reboot the server::
 
 PuTTY will disconnect when the server reboots.
 
-While this setup includes basic steps to protect your server against
-attacks, much more can be done. In particular, `authenticating with a public key <https://help.ubuntu.com/community/SSH/OpenSSH/Keys>`_
+.. note::
+
+Authenticating with a public key <https://help.ubuntu.com/community/SSH/OpenSSH/Keys>`_
```

Comment:
The lines of text to be called out need to be indented so they can be parsed properly (see [this example](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html) or check the structure of one of the other callouts [on this page](https://raw.githubusercontent.com/rmdash/docs/patch-7/masternodes/setup.rst)).

Also, restructured text URL processing is [a little different](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#links-to-external-web-pages), so the first link here isn't work due to a missing "`" to indicate the start of the link's text .